corehq/apps/userreports/tests/test_report_builder.py

```diff
@@ -136,7 +137,7 @@ def test_reference_for_forms(self):
             'form.case.update.last_name', "count",
         ]
 
-        self.assertEqual(expected_property_names, list(reference.data_source_properties.keys()))
+        self.assertItemsEqual(expected_property_names, six.iterkeys(reference.data_source_properties))
```

Comment:
Passing an iterable to `assertItemsEqual` seems like it could yield surprising results, depending on how it is implemented. I think this should be more predictable:
```suggestion
        self.assertItemsEqual(expected_property_names, list(reference.data_source_properties))
```

Reply:
Updated